### PR TITLE
Add datavirtualization extension

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Moving from other repo to make this easier to maintain. This was originally started by an issue updating to the new telemetry package which required the extension to be webpacked - and in doing that I decided just to do the work and move it here since it's always been annoying to deal with it being in the other repo and there's no reason to not have it public now.

The backing service will stay in the other repo for now, although at some point I'd like to move that into the STS repo or somewhere similar too.

Mostly a direct copy/paste, but there's a lot of cleanup I did (such as removing most of the package dependencies since they weren't actually being used anymore).

Tests currently aren't running, and there's likely runtime issues but I'm getting this first PR in so that follow up PRs to fix the other things can be more targeted and easily reviewed.

### CODE REVIEW

File: extensions/datavirtualization/config.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	"version": "1.10.0",
+	"downloadFileNames": {
+		"Windows_64": "win-x64.zip",
+		"Windows_86": "win-x86.zip",
+		"OSX": "osx-x64.tar.gz",
+		"Linux_64": "linux-x64.tar.gz",
+		"CentOS_7": "linux-x64.tar.gz",
+		"Debian_8": "linux-x64.tar.gz",
+		"Fedora_23": "linux-x64.tar.gz",
+		"OpenSUSE_13_2": "linux-x64.tar.gz",
```
Comment (aasimkhan30) [inline extensions/datavirtualization/config.json:12]: Didn't know that these platforms were supported.I should add this to import extension as well.

**Review by aasimkhan30 — APPROVED** [review]

File: extensions/datavirtualization/config.json (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/datavirtualization/config.json:12]: You don't need to - the linux config in the import extension will cover all the various Linux distros. I plan on changing this to something similar as one of my follow up tasks.

Comment (alanrenmsft) [reply extensions/datavirtualization/config.json:12]: @aasimkhan30 remember that I've explained to you about this? you only need to specify a specific configuration it needs to use a different build. :)

**Review by alanrenmsft — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
Follow up tasks : https://github.com/microsoft/azuredatastudio/issues/21603

**Pull request merged**